modules/living_spaces_event/src/Form/LivingSpaceEventInviteUsersForm.php

```diff
@@ -22,22 +21,28 @@ class LivingSpaceEventInviteUsersForm extends FormBase {
   protected $entityTypeManager;
 
   /**
-   * Constructs a LivingSpaceEventInviteUsersForm form.
+   * The mail manager.
    *
-   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
-   *   Provides an interface for entity type managers.
+   * @var \Drupal\Core\Mail\MailManagerInterface
    */
-  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
-    $this->entityTypeManager = $entity_type_manager;
-  }
+  protected $mailManager;
```

Comment:
we don't use mail manager and config factory for the form anymore. you can remove it.
in order words, you can leave changes related to the warning message. that's all we need to update.